src/open_inwoner/openzaak/documents.py

```diff
@@ -100,9 +101,9 @@ def upload_document(
     user: SimpleLazyObject,
     file: InMemoryUploadedFile,
     title: str,
-    user_choice: int,
+    informatieobjecttype_url: str,
```

Comment:
This function doesn't need to know about `ZaakTypeInformatieObjectTypeConfig`, it just wants the URL for the informatieobjecttype.